src/esbmc/CMakeLists.txt

```diff
@@ -39,10 +39,12 @@ target_include_directories(esbmc
     PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
     PRIVATE ${Boost_INCLUDE_DIRS}
 )
-
+cmake_path(SET MY_MAGIC_PATH NORMALIZE "${cvc5_DIR}/../../")
+cmake_path(ABSOLUTE_PATH MY_MAGIC_PATH OUTPUT_VARIABLE CVC_LINK_DIR)
+message("Looking at ${CVC_LINK_DIR}")
 target_link_libraries(esbmc ${OLD_FRONTEND_TARGETS} ${SOLIDITY_FRONTEND_TARGETS} ${PYTHON_FRONTEND_TARGETS}
                             ${GOTO_CONTRACTOR_TARGETS} ${JIMPLE_FRONTEND_TARGETS} ${JIMPLE_FRONTEND_TARGETS}
                             clangcfrontend clangcppfrontend filesystem symex pointeranalysis langapi util_esbmc bigint
                             solvers clibs gotoalgorithms cache ${Boost_LIBRARIES} goto2c)
-
+target_link_directories(esbmc PUBLIC "${CVC_LINK_DIR}")
```

Comment:
I don't know why this would be necessary. Maybe some workaround for the dynamic loader lookup paths on macos that @ibnyusuf mentioned? I'm inclined to remove it. @rafaelsamenezes What do you think?